.idea/workspace.xml

```diff
@@ -2,7 +2,8 @@
 <project version="4">
```

Comment:
I think you actually don't want the .idea folder in the repo as it's specifically for project settings in JetBrains products.

1) carefully move the .idea folder somewhere else
2) add a commit that deletes the folder from the repo
3) add a .gitignore (if we don't already have one) and a line with ```.idea```
4) move your .idea folder back

After that .idea shouldn't show up when you try to do a ```git status```. Note that you should do all that from the terminal.